CHANGES_UNRELEASED.md

```diff
@@ -2,4 +2,12 @@
 
 # Unreleased Changes
 
+## Android
```

Comment:
nit: I think the title here should be the name of the component, e.g. "Places" in this case. See also https://github.com/mozilla/application-services/pull/2314 which I just filed after noticing this here.

Reply:
Corrected, thanks!